htsinfer/htsinfer.py

```diff
@@ -310,6 +310,6 @@ def print(self):
         sys.stdout.write(
             self.config.results.json(
                 indent=3,
-                sort_keys=True,
+                sort_keys=False,
```

Comment:
Why did you do that?

Reply:
With the added read length stats, the sorted keys get reported alphabetically as: "max, mean, median, min, mode" which just looks weird to me, as the order they inferred is "min, max, mean, median, mode" :sweat_smile: Should I change it back?

Reply:
Is there some consistency in the sort order for subproperties for other inferences (children, like here)? And for the different inferences themselves (parents)? If so, we should stick to it, I think, regardless of how it looks. Principle of least surprise ;-)

Reply:
Actually there's none. When unsorted, the parent keys just get reported in the order they're inferred (which I kinda think makes more sense): `library_stats` (read length stats), `library_type` (relationship between read pairs), `library_source `(organism), `read_orientation`, `read_layout`, while when sorted, it is: `library_source`, `library_stats`, `library_type`, `read_layout`, `read_orientation`. Apart from the stats mentioned above (min, max etc), the children are the same. So I suppose if you deem the inferred organism the most important and put it first, it makes sense to sort it.

Reply:
Well, if there's no consistent sort order, then just leave it "unsorted" :)